.circleci/continue_config.yml

```diff
@@ -112,7 +112,6 @@ jobs:
               -DPRESTO_ENABLE_REMOTE_FUNCTIONS=ON \
               -DPRESTO_ENABLE_JWT=ON \
               -DCMAKE_PREFIX_PATH=/usr/local \
-              -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
```

Comment:
Why are we removing the ccache?

Reply:
I just noticed `[TEST_ONLY]`. My bad.

Reply:
> I just noticed `[TEST_ONLY]`. My bad.

@majetideepak I remove ccache for testing because I suspect that without ccache the failure is harder to surface. Will add it back later in a formal fix